src/domain/App.js

```diff
@@ -0,0 +1,46 @@
+const InputView = require("../view/InputView");
+const OutputView = require("../view/OutputView");
+const MESSAGES = require("../constant/Constant");
+const CarRace = require("./CarRace");
+const CarRaceGame = require("./CarRaceGame");
+
+class App {
+  #carRace = new CarRace();
+
+  play() {
+    this.#start();
+  }
+
+  async #start() {
+    try {
+      this.#carRace.setCarNames(await InputView.readCarNames(MESSAGES.carText));
```

Comment:
함수의 인자로 await 를 넘기는 것보다 구분을 해주는게 좋을 것 같아요!  
```suggestion
      const carNames = await InputView.readCarNames(MESSAGES.carText);
      this.#carRace.setCarNames(carNames);
```

Reply:
네 await로 값 받아올 때는 변수에 저장한 다음에 넘기도록 하겠습니다!